butte/src/error.rs

```diff
@@ -0,0 +1,35 @@
+#[derive(Debug, failure::Fail)]
```

Comment:
I think you should use the dtolnay's `anyhow` crate instead of `failure`  -- it's using the fixed `Error` trait and has a stable release

Reply:
Interesting, wasn't aware of that crate. Looks pretty sweet, I'll check it out.

Reply:
Interesting, wasn't aware of that crate. Looks pretty sweet, I'll check it out.

Reply:
Done.